index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lawrence's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/1a66ebfe2a.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <section>
+        <header>
+            <nav>
+                <div class="logo">My Logo</div>
+                <div>
+                    <i class="fa-solid fa-bars menu-bar"></i>
+                </div>
+
+
+            </nav>
+        </header>
+        <section class="introduction">
+            <h1 class="title"> I'm Lawrence Glad to See You</h1>
+            <div class="description">I’m a software developer! I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,don’t
+                hestiate to contact me.</div>
```

Comment:
- [ ] Well done on creating this section, however, it is required to use a `p` tag for this text, kindly modify this div into a `p`